research-software-directory/RSD-as-a-service
# Remove OAI-PMH table

# Remove OAI-PMH table

Changes proposed in this pull request:

* Remove the `oaipmh` table from the database, remove the endpoint from `nginx.conf` and remove the scraper from `jobs.cron`.

How to test:

* `docker compose down --volumes && docker compose build --parallel && docker compose up --scale data-generation=0`
* Visiting http://localhost/oaipmh and http://localhost/oaipmh?metadataPrefix=datacite4&verb=ListRecords should generate a frontend error page, as NGINX no longer sees this as a separate endpoint.

PR Checklist:
